src/usr/local/www/interfaces_ppps_edit.php

```diff
@@ -671,6 +680,13 @@ function build_country_list() {
 					'Check the "Configure NULL" box to configure a blank Service name.');
 
 	$section->add($group);
+
+	$section->addInput(new Form_Input(
+		'pppoe_host-uniq',
+		'Host-Uniq',
+		'text',
+		$pconfig['pppoe_host-uniq']
+	))->setHelp('This field can usually be left empty.');
```

Comment:
Needs a better description of what the field is and why someone might need to set it.